components.go

```diff
@@ -808,14 +801,14 @@ type VAlarm struct {
 	ComponentBase
 }
 
-func (c *VAlarm) Serialize() string {
+func (alarm *VAlarm) Serialize() string {
```

Comment:
In some cases your corrections have used an acronym (more than 2 words I gather) and in others the words. -- Intentional?

Reply:
I reworked it a bit. Using very short receiver names now, as is common practice. `c` for component, `p` for property. Then `cb` and `bp` up in their bases. And some other things.

Reply:
It's all good. I think we can start to develop a style guide from your work here. I like what you're doing though. It's good work! Thanks

However `cal` instead of `c` or `calendar` strikes me as nicer for the single word stuff.